debian/jitsi-meet-prosody.postinst

```diff
@@ -252,6 +245,27 @@ case "$1" in
         if [ "$PROSODY_CONFIG_PRESENT" = "false" ]; then
             invoke-rc.d prosody restart || true
 
+            # give it some time to warm up
+            sleep 10
```

Comment:
Can we use netcat or something to be sure?

Reply:
What do you mean? I'm not sure we can detect when all virtual hosts and stuff are loaded in prosody.

Reply:
Coming late to this, but maybe we could have an http handler in a module, and curl for it.  It might not tell us everything but at least it'd confirm the basics?